ntpd/src/ctl.rs

```diff
@@ -122,28 +117,10 @@ impl NtpCtlOptions {
             }
         }
 
-        options.resolve_action();
         // nothing to validate at the moment
 
         Ok(options)
     }
-
-    /// from the arguments resolve which action should be performed
-    fn resolve_action(&mut self) {
-        if self.help {
-            self.action = NtpCtlAction::Help;
-        } else if self.version {
-            self.action = NtpCtlAction::Version;
-        } else if self.validate {
-            self.action = NtpCtlAction::Validate;
-        } else if self.status {
-            self.action = NtpCtlAction::Status;
-        } else if self.force_sync {
-            self.action = NtpCtlAction::ForceSync;
-        } else {
-            self.action = NtpCtlAction::Help;
-        }
-    }
```

Comment:
Needs fix: it is actually quite subtle, but removing this, in combination with the changes above, actually changes behaviour, and in an unwanted way. In particular, it changes behaviour of the -v and -h flags overriding all other actions, which is undesired. Please revert.